### DISCUSSION TITLE
German letters!

### DISCUSSION THREAD

**Discussion by RantarR** [evennia/evennia#2534] [Evennia Questions] [answered]
Certain letters (ä,ö...) are not displayed correctly in evennia. How can I display them in my descriptions?

**Comment by Griatch** [comment, marked as answer]
Evennia itself does handle letters like åäö etc just fine - the entire server uses UTF-8 (I should know, I'm Swedish). More likely it's your MUD- client that does not support it or is using the wrong encoding. Switch to UTF-8 and you should be fine.
